packages/state-transition/src/slot/upgradeStateToCapella.ts

```diff
@@ -55,16 +55,7 @@ export function upgradeStateToCapella(stateBellatrix: CachedBeaconStateBellatrix
     currentVersion: config.CAPELLA_FORK_VERSION,
     epoch: stateBellatrix.epochCtx.epoch,
   });
-
-  // Upgrade the validators, this validator change is not present in latest specs but in 1.2.0
-  // so just set it to 0 and cleanup later
-  for (let i = 0; i < stateCapella.validators.length; i++) {
-    const validator = stateCapella.validators.get(i);
-    validator.fullyWithdrawnEpoch = Infinity;
-  }
-  // Nothing to do for latestExecutionPayloadHeader as the root is set to 0 by default
-  stateCapella.withdrawalQueue = ssz.capella.WithdrawalQueue.defaultViewDU();
-  // nextWithdrawalIndex and nextPartialWithdrawalValidatorIndex are also set to 0 by default
+  // nextWithdrawalIndex and latestWithdrawalValidatorIndex are already set to 0 by default
```

Comment:
Comment table above is stale withdrawalQueue still declared

Reply:
Add new line above ` // nextWithdrawalIndex and latestWithdrawalValidatorIndex are already set to 0 by default` comment